app/ui/archiveTile.js

```diff
@@ -169,6 +164,19 @@ module.exports = function(state, emit, archive) {
             <img src="${assets.get('share-24.svg')}" class="mr-2" /> Share
           </button>
         `;
+  const dl =
+    platform() === 'web'
```

Comment:
I think the download link would be just as useful on mobile. A good way to get a file from a desktop machine to a mobile device?

Reply:
interesting. should we add the download ui to the android app then?

Reply:
It could just be a link to the server.